### PR TITLE
link to incidents on first reference

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/developer-experience/issues/385 -- links to incidents on first reference.

🥳 I think(?) this is our first resolved PushFeedback-related issue??

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: versioned_docs/version-8.5/components/modeler/bpmn/error-events/error-events.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ event or error event subprocess that caught the error is activated.
 
 ## Unhandled errors
 
-When an error is thrown and not caught, an **incident** (i.e. `Unhandled error event`) is raised to indicate the failure. The incident is attached to the corresponding element where the error was thrown (i.e. the task of the processed job or the error end event).
+When an error is thrown and not caught, an [**incident**](/components/concepts/incidents.md) (i.e. `Unhandled error event`) is raised to indicate the failure. The incident is attached to the corresponding element where the error was thrown (i.e. the task of the processed job or the error end event).
```
Comment (mesellings) [inline versioned_docs/version-8.5/components/modeler/bpmn/error-events/error-events.md:62]:
```suggestion
When an error is thrown and not caught, an [**incident**](/components/concepts/incidents.md) (for example, `Unhandled error event`) is raised to indicate the failure. The incident is attached to the corresponding element where the error was thrown (that is, the task of the processed job or the error end event).
```
Just a suggestion, spotted these i.e. instances

**Review by mesellings — APPROVED** [review]
Lgtm, great idea 👍 - just one comment for unrelated instances of i.e. we could tackle whilst we're in the doc?

File: versioned_docs/version-8.5/components/modeler/bpmn/error-events/error-events.md (hunk 1 above)
Comment (christinaausley) [reply versioned_docs/version-8.5/components/modeler/bpmn/error-events/error-events.md:62]: Good little catch along the way 🧹

**Review by mesellings — APPROVED** [review]
Lgtm! 👍 🚀

**Pull request merged**